sw/device/rom_exts/manifest.md

```diff
@@ -245,7 +245,7 @@ Notes:
 1.  **Code Image** This is a sequence of bytes that make up the code and data of
     the ROM_EXT image. This data will include any data required for Extensions.
 
-    **Execution begins** at the address 128 (`0x80`) bytes beyond the first
+    Execution begins at the address 128 (`0x80`) bytes beyond the first
```

Comment:
Maybe it is worth explaining the rationale why we keep the jump into `rom_ext_start_` inside the interrupt vector, even though it is not actually hit on `power-on`, but is jumped to from the Mask ROM?

Reply:
As I said in my meeting, I'm not 100% sure we'll set `mtvec` before jumping into ROM_EXT, so I want this document not to commit us to that yet.

Do you have concrete suggestions for this paragraph, to clarify "This matches how Ibex boots."? Note whether we pretend this is part of the vector is strictly an implementation detail - this specification of behaviour does not require it, I just think it's a better implementation approach.

Reply:
Sure, makes sense